test/python/circuit/test_circuit_load_from_qasm.py

```diff
@@ -54,14 +53,37 @@ def test_qasm_file(self):
         q_circuit_2.measure(qr_b, cr_d)
         self.assertEqual(q_circuit, q_circuit_2)
 
+    def get_qelib1_gates(self):
+        """
+        Get all the gates from qelib1.inc library as a set of strings
+        """
+        qelib1_path = (Path(qiskit.qasm.__file__).parent / "libs/qelib1.inc").absolute()
```

Comment:
You should do `Path / "libs" / "qelib1.inc"` if you want to be safe; the form here implies that `/` is the file separator, which is the exact problem that both `os.path` and `pathlib` are designed to abstract, since it's not universally true.